### DISCUSSION TITLE
Stream on Apple Watch

### DISCUSSION THREAD

**Discussion by bellario89** [mrlt8/docker-wyze-bridge#407] [Q&A] [answered]
Hi this project has been amazing but I’m able to stream and record with Scrypted on my Synology from my all of my devices except for my watch. Has anyone got this to work? We have three Wyze V3 cams and this is the only hiccup I’ve had. I’ve tried transcoding the low res stream to no avail.

**Comment by cheme75** [comment]
I got the bridge going to scrypted as rtsp cams and all seem to stream to watch, iphone, ipad, tv as long as the bridge and scrypted machines are running - these machines are over 10 yrs old and pretty limited so occasionally lock up but overall they run well enough for present needs. I use the bridge to rotate the DB since this machine has more processing ability and it streams well. I transcode and traanspose one cam in scrypted and its slow and jerky - wish I could rotate it in the bridge - would be much smoother at least until I improve my machines

**Reply by bellario89** [reply]
Hmmm I can stream everywhere but my watch

**Comment by cheme75** [comment, marked as answer]
Wish I knew more to advise - it’s working here. I also just noticed it’s only the snapshot in landscape for the rotated cam, when it streams that cam, it rotates correctly. Anyway, this is the doorbell, it has a better image today with all the rain hazing up the windows a bit where the cams are. 

![292CBFE3-0C59-4CC0-8B79-4FB434E49073](https://user-images.githubusercontent.com/16415255/172023493-d7f5a2e8-668a-4a35-a03c-f840e65b1719.jpeg)

**Reply by bellario89** [reply]
Can you share you sensitive env-variables for the bridge?

**Comment by cheme75** [comment]
Pretty basic. There’s the 3 ports, set user and pw. I don’t have remote access to the yml file from here but it’s pretty basic. I set time zone, ignore offline, name the 3 cams to filter for just them, then enable audio for db cam, set codec to aac, rotate door and rotate one cam. Can’t think of anything else.  I keep trying LAN only mode but it keeps failing so I comment it out until I think of something else. Scrypted uses the 3 RTSP streams set as cams with the recommended plugins all pretty much using defaults other than tweaking the cv plugin settings for motion detection. I’m not using iCloud+ to record or do ai detection on any yet.

**Reply by bellario89** [reply]
What's your resolution settings for them ? 120HD?

**Comment by cheme75** [comment]
Didn’t set it in env. Whatever it defaults.
